src/util.js

```diff
@@ -49,7 +49,7 @@ export function plugins ({ pkg, deps, blacklist = []}) {
   return deps
     .filter((dep) => /^fly-.+/g.test(dep))
     .filter((dep) => !~blacklist.indexOf(dep))
-    .reduce((prev, curr) => [].concat(prev, curr))
+    .reduce((prev, curr) => [].concat(prev, curr), [])
```

Comment:
It's fixed, sorry, but don't worry, please keep contributing.

You were mentioned in the CHANGELOG.md as well. :+1: